deny.toml

```diff
@@ -157,6 +162,14 @@ license-files = [
 unknown-git = "deny"
 unknown-registry = "deny"
 allow-git = [
+    # This needs Ord/Eq implementations hacked in so that we
+    # can store its structures in a UnaryFunc.
+    # The implementations are not semantically meaningful,
+    # (and involve disabling an _actually_ semantically meaningful
+    # PartialOrd impl)
+    # so it's unlikely they'd be accepted upstream.
+    "https://github.com/umanwizard/levenshtein-automata.git",
```

Comment:
I'm a bit nervous about a fork that never has a chance of going back upstream. Since you store the actual needle in the function struct, what about just ignoring the automata field in the `PartialOrd`/`Ord`/`Eq`/`PartialEq` implementations? (You can use the `derivative` crate to make this less painful.)

Reply:
>But I couldn't figure out where we actually do that, except in tests. Do you happen to know?

Probably sending over a TCP connection in distributed mode! In which case doing it on every worker once is not the end of the world. But also it seems like `Serialize/Deserialize` is something that could, plausibly, be accepted upstream, to allow us to unfork?

Reply:
> Why don't you like is_approx_match, is it because it's not explicit? We could do something like is_dev_lam_within if you like that better.

Yeah, "approx" is a new abbreviation we're inventing, and if we did eventually add `levenshtein` for pgcompat then `is_approx_match` would not be obviously related. What about `is_levenshtein_less_equal`/`is_levenshtein_damerau_less_equal`?